test/Microsoft.Azure.WebJobs.Host.TestCommon/TestConfigurationBuilderExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Azure.WebJobs.Host.TestCommon
+{
+    public static class TestConfigurationBuilderExtensions
+    {
+        /// <summary>
+        /// Allows configuration to come from %userprofile%\.azurefunctions\appsettings.tests.json
+        /// </summary>
+        /// <param name="builder">The configuration builder.</param>
+        /// <returns>The modified configuration builder.</returns>
+        public static IConfigurationBuilder AddTestSettings(this IConfigurationBuilder builder)
+        {
+            string configPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), ".azurefunctions", "appsettings.tests.json");
+            return builder.AddJsonFile(configPath, true);
+        }
```

Comment:
Added this in a bunch of places. Made testing variations manually a ton quicker.

Reply:
Nice to take advantage of this in tests. We need to document the ability to do this so devs are aware as this is super helpful.